src/transformers/trainer.py

```diff
@@ -2175,6 +2175,9 @@ def _inner_training_loop(
         grad_norm: Optional[float] = None
         self.control = self.callback_handler.on_train_begin(args, self.state, self.control)
 
+        if args.sanity_evaluation:
+            self._evaluate(trial, ignore_keys_for_eval)
```

Comment:
```suggestion
            self._evaluate(trial, ignore_keys_for_eval, True)
```

Reply:
Done ! but I also added the arg name for readability